inputfiles/idl/DOM.widl

```diff
@@ -214,7 +214,7 @@ interface Node : EventTarget {
   readonly attribute Document? ownerDocument;
   Node getRootNode(optional GetRootNodeOptions options = {});
   readonly attribute Node? parentNode;
-  readonly attribute Element? parentElement;
+  readonly attribute HTMLElement? parentElement;
```

Comment:
Is this a change from upstream?

Reply:
Thanks for catching this! Happened during a RegExp replace operation..